.github/workflows/ios_manual.yml

```diff
@@ -92,7 +101,8 @@ jobs:
           xcrun simctl shutdown all
           export SIMULATOR_UDID=$(python get_specific_device_udid.py)
           poetry run python generate_smoke_tests.py
-          poetry run pytest --experiment ${{ inputs.slug }} --experiment-branch ${{ matrix.branch }} --feature ${{ inputs.feature-name }} ${{ inputs.extra-arguments }}
+          echo "TEST COMMAND: poetry run pytest --experiment ${{inputs.experiment-server}}/${{ inputs.slug }} --experiment-branch ${{ matrix.branch }} --feature ${{ inputs.feature-name }} ${{ inputs.extra-arguments }}"
```

Comment:
How come the formatting is different here than the Android one? Ie server/slug

Reply:
Oh good catch, this shouldn't be in this PR